### PR TITLE
Commerce 10847 3

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-8364

change requests from here https://github.com/brianchandotcom/liferay-portal/pull/135735

### CODE REVIEW

File: modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AccountSelectorTag.java [hunk 1]
```diff
@@ -75,20 +75,12 @@ public int doStartTag() throws JspException {
...
-
 			_accountEntryAllowedTypes =
 				commerceContext.getAccountEntryAllowedTypes();
-
-			_themeDisplay = (ThemeDisplay)httpServletRequest.getAttribute(
-				WebKeys.THEME_DISPLAY);
-
-			_addCommerceOrderURL = _getAddCommerceOrderURL(httpServletRequest);
-
 			_accountEntry = commerceContext.getAccountEntry();
```
Comment (smottal) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AccountSelectorTag.java:80]: It should go before _accountEntryAllowedTypes

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**